src/app/globals.css

```diff
@@ -23,11 +23,127 @@
 body {
   background: var(--background);
   color: var(--foreground);
-  font-family: "Mulish", Arial, Helvetica, sans-serif;
+  font-family: "Open Sans", Arial, Helvetica, sans-serif;
 }
 
-.mulish-uniquifier{
-  font-family: "Mulish", sans-serif;
+.open-sans-ifdm {
+  font-family: "Open Sans", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 400;
+  font-style: normal;
+  font-variation-settings:
+    "wdth" 100;
+}
+
+.poppins-thin {
```

Comment:
Hi @jenbreese 

Why add all these lines of code when we can do the same thing by importing 'Poppins' the same way we import Open Sans?

Remove lines 38 through 144 and import poppins how [NextJS docs say to import fonts](https://nextjs.org/docs/pages/getting-started/fonts). Then for each tag you want to use Poppins, update the class use something like this `<h2 className={ `${poppins.className} font-bold` }>